packages/semcom-node/package.json

```diff
@@ -34,20 +34,18 @@
     "lint:fix": "eslint --fix .",
     "prepare": "npm run build",
     "start": "node dist/main.js",
-    "test": "jest",
-    "test:ci": "jest --runInBand"
+    "test": "jest --silent ",
+    "test:ci": "jest --runInBand --silent"
```

Comment:
Ha, we should definitely decide on a convention for this, because everyone is changing these all the time. An issue for dgt-docs ...